docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/03-Digital-Ocean-(manual).md

```diff
@@ -157,14 +193,14 @@ Changes:
   + Created field `updatedAt` of type `DateTime!`
   + Created field `createdAt` of type `DateTime!`
 
-Applying changes 1.8s
+Applying changes 1.2s
 
-Hooks:
+Your Prisma GraphQL database endpoint is live:
 
-Running $ graphql prepare...
+  HTTP:  http://37.139.15.166:4466
+  WS:    ws://37.139.15.166:4466
+```
 
-Your Prisma database service endpoint is live:
+Connect to the `/management` endpoint in a browser. If your droplet has the ip `37.139.15.166` open the following webpage: `http://37.139.15.166:4466/management`. If a GraphQL Playground shows up, your Prisma server is set up correctly.
```

Comment:
"For example, if your Droplet has the ip `37.139.15.166` open the following webpage: `http://37.139.15.166:4466/management`."